src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+body {
+  margin: 0;
+  font-family: $font-family;
+}
+
+.card {
+  border-radius: $border-radius;
+  height: $card-height;
+  width: $card-width;
+  border: 1px solid $border-color;
+  color: $card-background;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  &__image {
+    width: $image-width;
+    height: $image-hight;
+    display: block;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: $title-font-size;
+    font-weight: 500;
+    color: $text-main-color;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__code {
+    font-size: $basic-font-size;
+    color: $text-secondary-color;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0 84px 16px 16px;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 16px 24px;
+
+    &-rate {
+      font-size: $basic-font-size;
+      color: $text-main-color;
```

Comment:
Could you try to center this text? By comparing it to figma/tests you can see that in your solution it shifts up